### PR TITLE
Testing todo list App

### PR DESCRIPTION
**Pull request by blase147**
Testing todo list App

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#   *Status: Invalidated* :red_circle: 


Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that reflect commit from one student. This is a pair-programming project. The two partners must have at least a commit in the PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by blase147** [comment]
Thanks for the review. However, my partner has been away and is scarcely available due to personal issues. I had sent a message on zendesk concerning this. Even now I have exhausted all channels of reaching him all to no avail. I don't know what to do

File: dist/index.html [hunk 1]
```diff
@@ -3,36 +3,29 @@
...
+            <section>
+                <div class="new-task-field">
+                    <input type="text" placeholder="Add to your List">
+                </div>
+                <ul class="tasks">
+                </ul>
+                <button class="clearAll">Clear all completed</button>
+            </section> 
+        </article>
+        <p></p>
```
Comment (Nowembabazi) [inline dist/index.html:28]: optional changes. Could you please clear these from your code if they are empty or if they don't entail any material.

**Review by Nowembabazi — APPROVED** [review]
##Halo Paul,
I want to recommend the excellent work done so far on the project, 
-Perfect commit messages
-Good workflow
-Proper use of linters.
-Proper use of javascript syntax
-Good use of HTML syntax.

You can now merge your work now the project has been approved.
##Otherwise good work on the project. 
Good work and happy coding.

**Pull request merged**